uber/templates/signups/shifts_old.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
```

Comment:
Is this file not actually used?  If I'm understanding things correctly, then the new stuff replaces this.  In general, we shouldn't leave old files sitting around in our repo because getting them back later is what source control is for.